vignettes/dm-class-and-basic-operations.Rmd

```diff
@@ -35,19 +35,18 @@ Let's first have a look at the new class:
 The `dm` class contains a set of tables as well as information about their primary keys and the relations between the tables. 
 You can have your `dm` object on different data sources, such as a local environment or a database.
 
-`dm` is a wrapper around two classes, which are independent from each other:
-
-1. part of the object is a [`src` object](https://dplyr.tidyverse.org/reference/src.html) from {dplyr}
-2. the other part is a class `data_model` object
+`dm` is a wrapper around the two `data_model` object
```

Comment:
We're moving away from this. It's more important to understand that we handle tables, relationships, and now also filters.

The text works "as is", let's wait until #57 before revisiting.